FetchingFriends/fetching-friends-ui
# Feature/login

## What does this PR do?
- It updates the landing page with styling and buttons
## How should it be tested in the UI?
- Open the landing page in your browser and make sure everything is displaying correctly.
